backend/dataall/modules/dataset_sharing/services/share_item_service.py

```diff
@@ -63,7 +63,7 @@ def revoke_items_share_object(uri, revoked_uris):
                 resource_uri=dataset.datasetUri,
             )
 
-            ShareNotificationService.notify_share_object_rejection(session, context.username, dataset, share)
+            ShareNotificationService.notify_share_object_rejection(session, context.username, context.email_id, dataset, share)
```

Comment:
Very similar to the inital definition of email_id in context. Is there a difference in other IdPs between the username and the email?

Reply:
I created this new context field so as to have clear usage of what is being passed. At that time I wanted to rename username to email_id but that would have added some more refactoring into entire data.all backend code where the `username` field is used.